requirements.txt

```diff
@@ -5,12 +5,15 @@ sqlalchemy >= 1.3
 https://github.com/lsst/sphgeom/archive/master.tar.gz
 
 # optional
+backoff >= 1.10.0
```

Comment:
Do you need the specifications to be this precise? Typically minor version is sufficient, no?

Reply:
Changed to minor version.